### PR TITLE
[MRG] Add recipe for mz_to_sqlite

### PR DESCRIPTION
**Pull request by souravsingh**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by souravsingh** [comment]
@bgruening I need some help in reviewing the recipe.

**Comment by daler** [comment]
The current master branch appears to be broken, and we'll fix that. But a separate problem is that the `source: fn` and `source: url` are empty in this PR.

**Comment by souravsingh** [comment]
About that, I wasn't sure about the fields since I couldn't find the link for appropriate zip file.

**Comment by bgruening** [comment]
@souravsingh try this one: https://github.com/galaxyproteomics/mzToSQLite/releases/download/1.2.0/mzToSQLite-1.2.0.jar

File: recipes/mz_to_sqlite/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+requirements:
+  build:
+  run:
+    - java-jdk  >=6
+    - python
+
+test:
+    commands:
+      -
```
Comment (bgruening) [inline recipes/mz_to_sqlite/meta.yaml]: You also should add a test case here, very much like the one from PS

**Comment by souravsingh** [comment]
@bgruening The tests for the recipe are failing. Did I write a wrong test?

**Comment by bgruening** [comment]
Yes, seems so: https://travis-ci.org/bioconda/bioconda-recipes/jobs/184904654#L792
`Command not found.`
I don't now much about this tool. But can you try: `mz_to_sqlite --help`

mz_to_sqlite is the python wrapper that calls the JAR file and passes all arguments to the JAR file. If the Java program understands `--help` this should work.

Please also keep in mind that you can test these thing locally with `conda build recipes/mz_to_sqlite/`

**Comment by souravsingh** [comment]
@bgruening The recipe is now ready for bioconda

**Comment by bgruening** [comment]
Awesome! Thanks!

**Pull request merged**